docs/src/main/paradox/user/lang/kotlin/gettingstarted.md

```diff
@@ -0,0 +1,288 @@
+                    <to>
+                        <image>${repo.name}</image>
+                        <credHelper></credHelper>
+                        <tags>
+                            <tag>${tag.version}</tag>
+                        </tags>
+                    </to>
+                    <container>
+                        <mainClass>${main.class}</mainClass>
+                        <jvmFlags>
+                            <jvmFlag>-XX:+UseG1GC</jvmFlag>
+                            <jvmFlag>-XX:+UseStringDeduplication</jvmFlag>
+                        </jvmFlags>
+                        <ports>
+                            <port>8080</port>
+                        </ports>
+                    </container>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    
+</project>
+```
+@@@
+
+Subsequent source locations and build commands will assume the above Maven project, and may need to be adapted to your particular build tool and setup.
+
+## Cloudstate CLI example
+
+If you are a Linux or MacOS user and have [Cloudstate CLI](https://github.com/sleipnir/cloudstate-cli) version 0.5.0 or higher installed, you can generate the project scaffold with the commands below:
+
+@@@vars
+```shell script
+cloudstate init && \
+cloudstate create \
+  --name=shopping-cart \
+  --profile=kotlin \
+  --registry=docker.io/some-repo \
+  --set-user=some-user \
+  --set-pass=***** \
+  --tag=1.0.0 \
+  --set-editor=idea
+```
+@@@
+
+The above command will create the directories, generate the artifacts declared in the proto files and build the project for you. 
+
+## Protobuf files
+
+The Xolstice Maven plugin assumes a location of `src/main/proto` for your protobuf files. In addition, it includes any protobuf files from your Kotlin dependencies in the protoc include path, so there's nothing you need to do to pull in either the Cloudstate protobuf types, or any of the Google standard protobuf types, they are all automatically available for import.
+
+So, if you were to build the example shopping cart application shown earlier in @ref:[gRPC descriptors](../../features/grpc.md), you could simply paste that protobuf into `src/main/proto/shoppingcart.proto`. You may wish to also define the Kotlin package, to ensure the package name used conforms to Kotlin package naming conventions:
+
+```proto
+option java_package = "com.example.shoppingcart";
+```
+
+Now if you run `mvn compile`, you'll find your generated protobuf files in `target/generated-sources/protobuf/java`.
+
+## Creating a main function
+
+Your main class will be responsible for creating the Cloudstate gRPC server, registering the entities for it to serve, and starting it. To do this, you can use the `CloudState` function server builder, for example:
+
+@@snip [Main.kt](/docs/src/test/kotlin/docs/user/gettingstarted/Main.kt) { #shopping-cart-main }
+
+We will see more details on registering entities in the coming pages.
+
+## Parameter injection
```

Comment:
The table looks like this in my browser (maxOS Safari).
<img width="1310" alt="Screenshot 2020-03-04 at 23 50 31" src="https://user-images.githubusercontent.com/12616445/75930505-0072e080-5e73-11ea-8fbd-79984772644a.png">